### PR TITLE
NT-0043 - Decode token on client side / save tokenData in 1 field / alert fix

### PR DESCRIPTION
**Pull request by kamilmoskal**

### CODE REVIEW

File: client/src/components/Alert/Alert.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const Alert: React.FC<AlertProps> = ({
         horizontal
       }}
       open={open}
-      autoHideDuration={autoHideDuration}
+      autoHideDuration={open ? autoHideDuration : undefined}
```
Comment (mras2303) [inline client/src/components/Alert/Alert.tsx]: Please, explain me this ternary operator in comment and tell me if using && operator is impossible for this case

`autoHideDuration={open && autoHideDuration}`

Comment (kamilmoskal) [reply client/src/components/Alert/Alert.tsx]: @mras2303 
i've try many things but only this works :D
I did this becouse i noticed that CLER_ALERT actions i dispatching sometimes randomly

Comment (kamilmoskal) [reply client/src/components/Alert/Alert.tsx]: `autoHideDuration={open && autoHideDuration}`
i tried but i got error from types that this prop can't be false

File: client/src/store/actions/authActions.tsx [hunk 2]
```diff
@@ -105,12 +107,12 @@ export const logoutUser = (): LogoutAction => {
...
 };
 
-const setToken = (token: string, expTime: number) => (
+const setToken = (token: string) => (
   dispatch: (arg: Function) => void
 ) => {
-  localStorage.setItem("token", token);
-  const expDate = new Date(new Date().getTime() + expTime * 1000);
-  localStorage.setItem("expDate", JSON.stringify(expDate));
+  const decoded: {exp: number} = jwt_decode(token);
```
Comment (damian0o) [inline client/src/store/actions/authActions.tsx]: I believe we can only store token and decode it on demand

**Pull request merged**